infinispan-remote/cross-site-replication/docker-compose/docker-compose.yaml

```diff
@@ -9,38 +9,42 @@ services:
     ports:
       - "11222:11222"
     container_name: ispn-lon-1
+    environment:
+      USER: admin
+      PASS: password
     networks:
       - mynetwork
-    volumes:
-      - ${PWD}:/user-config
-    command: -b SITE_LOCAL -c /user-config/infinispan-xsite.xml -Djgroups.cluster.mcast_addr=228.6.7.10 -Drelay.site=LON
+    command: -c infinispan-xsite.xml -Dinfinispan.site.name=LON -Djgroups.cluster.mcast_addr=228.6.7.10
```

Comment:
you need to set different ports for different sites. for example, in LON use `-Djgroups.mcast_port=46656` and in NYC use `-Djgroups.mcast_port=46666`

Reply:
thanks!